assets/js/components/wp-dashboard/WPDashboardIdeaHub.js

```diff
@@ -41,36 +39,18 @@ import {
 import { trackEvent } from '../../util';
 import GoogleLogoIcon from '../../../svg/logo-g.svg';
 import Link from '../Link';
+import whenActive from '../../util/when-active';
 
 const { useSelect } = Data;
 
 function WPDashboardIdeaHub() {
-	const isIdeaHubEnabled = useFeature( 'ideaHubModule' );
-
-	const { hasSavedIdeas, isModuleActive, dashboardURL } = useSelect(
-		( select ) => {
-			if ( ! isIdeaHubEnabled ) {
-				return {};
-			}
-
-			const isActive = select( CORE_MODULES ).isModuleActive(
-				'idea-hub'
-			);
-			if ( ! isActive ) {
-				return {};
-			}
-
-			const savedIdeas = select( MODULES_IDEA_HUB ).getSavedIdeas();
-			const adminURL = select( CORE_SITE ).getAdminURL(
-				'googlesitekit-dashboard'
-			);
-
-			return {
-				isModuleActive: isActive,
-				hasSavedIdeas: savedIdeas?.length > 0,
-				dashboardURL: `${ adminURL }#saved-ideas`,
-			};
-		}
+	const savedIdeas = useSelect( ( select ) =>
+		select( MODULES_IDEA_HUB ).getSavedIdeas()
+	);
+	const dashboardURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard', {
+			'idea-hub-tab': 'saved-ideas',
```

Comment:
This fixes the URL used here which was missed after it was changed to use a query param instead.